README.md

```diff
@@ -1,4 +1,4 @@
-# [AtlasDB](https://palantir.github.io/atlasdb/) [![Download](https://api.bintray.com/packages/palantir/releases/atlasdb/images/download.svg)](https://bintray.com/palantir/releases/atlasdb/_latestVersion)
+# [AtlasDB](https://palantir.github.io/atlasdb/) - [Download the client](https://search.maven.org/artifact/com.palantir.atlasdb/atlasdb-client)
```

Comment:
nit: it may be useful to point to `atlasdb-api` as well as the client. The API contains `KeyValueService`, `Transaction` and various other essentials.